directus/app
# Fix #1632 — Unmanage collections loading icon

closes #1632 

This fixes the loading icon persisting when un-managing multiple collections. — I'm confident in the fix. :) 

I make a copy of the variable, as we don't want the reactive state being cleared, leaving us unable to unset the collection name from the array I'm using to manage the states due to an unassigned variable because it was wiped. :) 

**To test if needed:**
Click "Unmanage", confirm action, it should show loading icon, then close the popup, and display the button as "Manage".
Click "Unmanage" on another - loading icon should not be shown, click confirm action, it should show loading icon, then close the popup, and display the button as "Manage".

**Also, do the same, but close the modal early.**
Click unmanage, confirm, it shows loading icon, then click Cancel (or the dark-area) to close the modal, it should still unmanage the item (due to taking a copy of the variable).
